src/Models/transactionCode.model.ts

```diff
@@ -0,0 +1,7 @@
+import { Model } from 'objection';
+
+export default class Transaction extends Model {
+  static get tableName() {
+    return 'transactions';
```

Comment:
We spoke about table name to be a constant imported and use. This is prone to error